README.md

```diff
@@ -116,12 +116,12 @@ curl -L 'https://raw.githubusercontent.com/filiparag/wikiman/master/Makefile' -o
 # Example for Linux: install Arch Wiki
 make -f ./wikiman-makefile source-arch
 sudo make -f ./wikiman-makefile source-install
-make -f ./wikiman-makefile clean
+sudo make -f ./wikiman-makefile clean
 
 # Example for BSD: install FreeBSD Documentation
-make -f ./wikiman-makefile source-fbsd local
+make -f ./wikiman-makefile source-fbsd
```

Comment:
Please add back `local` to this line, so I can merge the edit.

Reply:
done